_episodes/09-branches.md

```diff
@@ -261,190 +206,212 @@ $ git branch
 * experiment
   less-salt
   master
-```
 
-### Visualizing branches
+$ git graph
+
+* bf59be6 (HEAD -> less-salt) reduce amount of salt
+| * 6feb49d (experiment) maybe little bit less cilantro
+| * 7cf6d8c let us try with some cilantro
+|/
+* dd4472c (master) we should not forget to enjoy
+* 2bb9bb4 add half an onion
+* 2d79e7e adding ingredients and instructions
+```
 
-Here is a graphical representation of what we have created (the commit hashes will be different on your laptop):
+Here is a graphical representation of what we have created (the commit hashes
+will be different on your laptop):
 
 ![]({{ site.baseurl }}/img/gitink/git-branch-2.svg)
 
-Lets try to produce a visual representation of the branches using git commands.
+- Now switch to `master`.
+- Add and commit the following `README.md` to `master`:
 
-Try the following
+```markdown
+# Guacamole recipe
 
-```shell
-$ git log --all --graph --decorate --oneline --abbrev-commit
+Used in teaching Git.
 ```
 
-This was very nice way to visualise the branches and the commits. But the command has too many parameters and it is too
-long to type. Fortunately Git has a solution for this using aliases:
+Now you should have this situation:
 
 ```shell
-$ git config --global alias.graph "log --all --graph --decorate --oneline --abbrev-commit"
+$ git graph
+
+* fdc8490 (HEAD -> master) draft a readme
+| * bf59be6 (less-salt) reduce amount of salt
+|/
+| * 6feb49d (experiment) maybe little bit less cilantro
+| * 7cf6d8c let us try with some cilantro
+|/
+* dd4472c we should not forget to enjoy
+* 2bb9bb4 add half an onion
+* 2d79e7e adding ingredients and instructions
 ```
 
-Next time when we want this we will use the alias
+![]({{ site.baseurl }}/img/gitink/git-branch-3.svg)
 
-```shell
-$ git graph
-```
+And for comparison this is how it looks [on GitHub](https://github.com/bast/recipe/network).
 
 ---
 
 ## Merging branches
 
+**If you got stuck in the above exercises**:
```

Comment:
Or call a helper to un-stick it for you?  Though if it's a merge conflict it's not easy to fix... so probably never mind.

Reply:
Yes I was wondering whether this is helpful or a trap for those who will execute it :-) Should I perhaps remove that part? It's just that a couple of times participants got stuck or created branches "wrong" and then could not type along in the merging part.

Reply:
I thought this was a perfectly clever way to save people!  There are plenty of warnings, so by all means leave it.